### ISSUE TITLE
[BUG] Error when choosing an already existant folder (with icon) for icon pack folder

### ISSUE THREAD

**Issue by Mara-Li** [FlorianWoelki/obsidian-icon-folder#85]
**Describe the bug**
The plugin can't save when I try to reuse an already existent folder for icon pack. 

**To Reproduce**
1. Create an icon folder with icons pack
2. Try to change the settings to this folder

**Expected behavior**
Saving the settings

**Screenshots**

Here is the log : 
> Uncaught (in promise) Error: EPERM: operation not permitted, rmdir 'N:\Seed\.obsidian\plugins\obsidian-icon-folder\icons'

**Comment by FlorianWoelki** [comment]
I cannot reproduce this issue. Can you try to describe the reproduction steps a bit more?

**Comment by Mara-Li** [comment]
What I do : 
1. Plugin installation ; Blank option. 
2. Installation of some icons 
3. Move the folder (and their content) to another location (here, from `.obsidian/obsidian-icon-folder/icons` to `.obsidian/icons`
4. In plugin setting, set the new location.
5. See error : 
```log
Uncaught (in promise) Error: ENOENT: no such file or directory, copyfile 'D:\Documents\Github\test_plugin\.obsidian\plugins\obsidian-icon-folder\icons\font-awesome-brands\42-group.svg' -> 'D:\Documents\Github\test_plugin\.obsidian\icons\font-awesome-brands\42-group.svg'
Promise.then (async)
step @ plugin:obsidian-icon-folder:30
fulfilled @ plugin:obsidian-icon-folder:28
Promise.then (async)
step @ plugin:obsidian-icon-folder:30
eval @ plugin:obsidian-icon-folder:31
__awaiter @ plugin:obsidian-icon-folder:27
eval @ plugin:obsidian-icon-folder:1416
(anonymous) @ app.js:1
```

**Comment by FlorianWoelki** [comment]
Should be fixed with [version 1.3.3](https://github.com/FlorianWoelki/obsidian-icon-folder/releases/tag/1.3.3).
